truffle/src/main/java/org/jruby/truffle/nodes/ext/BigDecimalNodes.java

```diff
@@ -178,175 +217,327 @@ public static boolean isNan(RubyBasicObject value) {
             return getBigDecimalType(value) == Type.NAN;
         }
 
+        private void setupCreateBigDecimal() {
+            if (createBigDecimal == null)
+                createBigDecimal = insert(CreateBigDecimalNodeFactory.create(getContext(), getSourceSection(), null, null));
```

Comment:
Yes, curly braces please. Our coding style comes from the Graal CheckStyle rules, which we don't enforce anymore but would still like to vaguely follow.